ws-service/types.ts

```diff
@@ -18,6 +18,17 @@ export interface SimplifiedTransaction {
   address: string
   rawMessage: string
   inputAddresses: string[]
+  prices: Array<{
+    price: {
```

Comment:
No need for all that. This is supposed to be simplified, so only take what is necessary, and for what I've gathered, it would be: `value` and `quoteId`.

Reply:
Then alter `getSimplifiedTransaction` to take out these fields.